openpgp/packet/signature.go

```diff
@@ -63,7 +63,7 @@ type Signature struct {
 
 	SigLifetimeSecs, KeyLifetimeSecs                        *uint32
 	PreferredSymmetric, PreferredHash, PreferredCompression []uint8
-	PreferredAEAD                                           []uint8
+	PolicyURI, PreferredAEAD                                []uint8
```

Comment:
This is quite nitpicky, but `PreferredAEAD` is related to the other three in the line above, while this one isn't. Could you put it on a separate line?
Also, `[]uint8` doesn't really make sense as a type here; while it's technically the same thing, the others are actual lists of numbers, this one isn't. It should probably be `string`, or at least `[]byte`.